explorer/explorerroutes.go

```diff
@@ -16,10 +16,10 @@ import (
 func (exp *explorerUI) Home(w http.ResponseWriter, r *http.Request) {
 	height := exp.blockData.GetHeight()
 
-	blocks := exp.blockData.GetExplorerBlocks(height, height-6)
+	blocks := exp.blockData.GetExplorerBlocks(height, height-5)
 
 	exp.NewBlockDataMtx.Lock()
-	exp.MempoolData.RLock()
+	exp.MempoolData.Lock()
```

Comment:
Why can't this be `RLock`?

Reply:
That slipped by, I was testing something. Changed back now.